src/e2e/puppeteer-environment.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable fp/no-class */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable no-console */
+const chalk = require('chalk')
+const JsDomEnvironment = require('jest-environment-jsdom')
+const puppeteer = require('puppeteer')
+const { setup: setupDevServer, teardown: teardownDevServer } = require('jest-dev-server')
+const fs = require('fs/promises')
+const path = require('path')
+
+/**
+ * Delay for tests.
+ */
+const delay = ms => new Promise(resolve => setTimeout(() => resolve(true), ms))
+
+/** */
+class PuppeteerEnvironment extends JsDomEnvironment {
+  constructor(config) {
+    super(config)
+  }
+
+  async setup() {
+    console.log(chalk.yellow('Setup Test Environment.'))
```

Comment:
Use `console.info` for informational logging and preserve `console.log` for debugging.